legacy/reqcounter/reqcounter.go

```diff
@@ -27,11 +27,13 @@ import (
 
 // RequestCounter records the number of HTTP requests to GCR.
 type RequestCounter struct {
-	Mutex      sync.Mutex    // Lock to prevent race-conditions with concurrent processes.
-	Requests   uint64        // Number of HTTP requests since recording started.
-	Since      time.Time     // When the current request counter began recording requests.
-	Interval   time.Duration // The duration of time between each log.
-	Resettable bool          // Reset after logging.
+	Mutex     sync.Mutex    // Lock to prevent race-conditions with concurrent processes.
+	Requests  uint64        // Number of HTTP requests since recording started.
+	Since     time.Time     // When the current request counter began recording requests.
+	Interval  time.Duration // The duration of time between each log.
+	Threshold uint64        // When to warn of a high request count during a logging cycle. Setting a
+	// non-zero threshold allows the request counter to reset each interval. Therefore, if left uninitialized,
```

Comment:
Nit: Useless use of `Therefore`

Reply:
Sorry I didn't see this earlier. Here's that quick fix: #386